### PR TITLE
[INTERNAL] Address some docs issues

### PR DESCRIPTION
**Pull request by ecker**
Some docs updates, e.g. fixing link to RFC 0006.

### CODE REVIEW

**Review by KlattG — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -39,4 +38,4 @@ UI5 Tooling consists of multiple modules. Each one is managed in a separate repo
 Please check our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/blob/main/CONTRIBUTING.md).
 
 ## Support
-Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/blob/main/CONTRIBUTING.md#report-an-issue) on how to report an issue. Or chat with us in the [`#tooling`](https://openui5.slack.com/archives/C0A7QFN6B) channel of the [OpenUI5 Community Slack](https://ui5-slack-invite.cfapps.eu10.hana.ondemand.com/). For public Q&A, use the [`ui5-tooling` tag on Stack Overflow](https://stackoverflow.com/questions/tagged/ui5-tooling).
+Please see how to [report an issue](https://github.com/SAP/ui5-tooling/blob/main/CONTRIBUTING.md#report-an-issue). Or chat with us in the [`#tooling`](https://openui5.slack.com/archives/C0A7QFN6B) channel of the [OpenUI5 Community Slack](https://ui5-slack-invite.cfapps.eu10.hana.ondemand.com/). For public Q&A, use the [`ui5-tooling` tag on Stack Overflow](https://stackoverflow.com/questions/tagged/ui5-tooling).
```
Comment (matz3) [inline README.md:41]: The first sentence sounds incomplete to me.

Please note that there's a very similar section in every module repo, e.g. https://github.com/SAP/ui5-builder#support

**Review by matz3 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (flovogt) [inline README.md:41]:
```suggestion
Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/blob/main/CONTRIBUTING.md#report-an-issue) on how to report an issue.

Please report issues [here](https://github.com/SAP/ui5-tooling/issues).
 
Chat with us in the [`#tooling`](https://openui5.slack.com/archives/C0A7QFN6B) channel of the [OpenUI5 Community Slack](https://join.slack.com/t/openui5/shared_invite/zt-1q128gn3p-JeZTi9XCpPxW8kBohSgqnw).

For public Q&A, use the [`ui5-tooling` tag on Stack Overflow](https://stackoverflow.com/questions/tagged/ui5-tooling).
```

File: docs/pages/Workspace.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ If a workspace configuration named `default` exists, it will be used automatical
 
 Workspaces can only be used in the project that is currently being worked on, i.e. the current root project. Workspace configurations of dependencies are ignored. The workspace file, however, does not necessarily have to be located within the project. You can reference any workspace configuration file via the UI5 CLI parameter `--workspace-config`.
 
-This concept has been discussed in an RFC: [RFC 0006 Local Dependency Resolution](https://github.com/SAP/ui5-tooling/blob/rfc-local-dependency-resolution/rfcs/0006-local-dependency-resolution.md)
+This concept has been discussed in an RFC: [RFC 0006 Local Dependency Resolution](https://github.com/SAP/ui5-tooling/blob/main/rfcs/0006-local-dependency-resolution.md)
```
Comment (matz3) [inline docs/pages/Workspace.md]: Fixed already

File: docs/pages/Troubleshooting.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ Please follow our [Contribution Guidelines](https://github.com/SAP/ui5-tooling/b
 ## UI5 Project
 ### Corrupt Framework Dependencies
 
-When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build- or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers like npm or Yarn.
+When using the UI5 CLI in versions lower than `v2.9.1` or the `@ui5/project` module in versions lower than `v2.2.6` you might experience build-time or runtime issues caused by missing or corrupt files in one or more UI5 framework dependencies. Note that this does not affect other project dependencies, such as reuse libraries, which are installed via external package managers like npm or Yarn.
```
Comment (matz3) [inline docs/pages/Troubleshooting.md]: Section was removed in v3

File: README.md (hunk 1 above)
Comment (matz3) [reply README.md]: I don't see a need to change this part.

**Review by flovogt — APPROVED** [review]

**Pull request merged**